doc/supported_file_formats.adoc

```diff
@@ -28,12 +28,12 @@ Minetest is able to read several types of file formats for use as assets in-game
 .3+| N/A
 
 | TGA (`.tga`)
-| Easy to programmatically encode and decode due to being a simple image format
+| Easy to programmatically encode and decode due to being a simple image format, and is to be preferred over BMP. Might be smaller than PNG at times due to a smaller file header.
```

Comment:
Suggestion: Replace “at times” with “for small or simple textures” and “due to a smaller file header” with “due to PNG having about 70 bytes overhead and TGA having about 20 bytes overhead” or something.

A 16×16 picture of one color is smaller as TGA than as PNG, but make it a checkerboard, then PNG wins.

Really, I have no idea how to actually phrase this in a matter that works well for a general audience.

Reply:
Imo the current formulation is fine.
Most people probably don't need to directly know here the minimum header and stuff size or that tga uses rle. Such things can be looked up elsewhere (or not at all, just export your texture in both formats and compare sizes).

Reply:
> just export your texture in both formats and compare sizes

A good idea in general. We absolutely have to document how to use OptiPNG then though.

For example, today I found 16×16 texture files in MineClone2 that were about 4500 bytes. Doing the math, even an uncompressed 16×16 RGBA image could have at most 16×16×4 bytes = 1024 bytes of payload. Turns out that it contained a lot of useless metadata that was added by GIMP. OptiPNG reduced the filesize by 97%!

Reply:
The usecase column is intended to be as brief as possible, just stating that the overhead is smaller which may lead to it being smaller at times should be enough.